gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/model/utils/MigrationResult.java

```diff
@@ -39,8 +41,9 @@ public MigrationResult(T value, Collection<Issue> issues) {
         this.issues.addAll(issues);
     }
 
+    @VisibleForTesting
     @Nullable
-    public T value() {
+    T value() {
```

Comment:
why removing the public here? especially to add an annotation that require to add an exception in the ArchiUnit test ensuring that we have a core depending on a carefully selected dependencies?